backend/src/MultiThreading/ExitHandlers.py

```diff
@@ -6,6 +6,10 @@
 from src.util.ErrorFactory import errorStackTrace
 from src.util.FileIO import FileIO
 
+SIGNALS_TO_IGNORE : [int] =[
```

Comment:
@xxdunedainxx I can remove these changes to ExitHandlers.py from this PR, correct?

Reply:
yah this shouldn't be in this PR, but can be file a github issue to add this into ExitHandlers in a separate PR?